### DISCUSSION TITLE
OLEDが点灯しない

### DISCUSSION THREAD

**Discussion by escape010** [Yowkees/keyball#298] [Q&A]
質問をお願い致します。

OLEDが点灯しません。
USB-CケーブルとPCを接続すると、pro micro内の赤いLEDが点灯し、
キーの動作確認はできますが、OLEDには何も表示されません。
（左右のOLED、どちらも同様の症状です）

尚、OLEDとピンを分離し、再度ハンダ付けをしようとしましたが、
ハンダ吸い取り線で、ハンダを除去しきれなかったのか、OLEDとピンを分離できませんでした。

お手数ですが、対処方法があれば、ご教示いただけるとありがたいです。
よろしくお願いいたいします。
![IMG_7830](https://github.com/Yowkees/keyball/assets/141485819/13921728-6186-4d49-ab23-7ee51258a048)
![IMG_7832](https://github.com/Yowkees/keyball/assets/141485819/d0f5a62f-118a-467e-8739-0d0ac2b20001)
![IMG_7833](https://github.com/Yowkees/keyball/assets/141485819/bb4e20cc-25fc-4ab5-bee7-d04b7275cd72)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

OLEDが点灯しないとのことで、その他のキー入力等の機能は問題ないでしょうか。

ひとまずProMicro裏の半田付けとジャンパーの半田付けでのミスが多いですので、ここから見直してみてください。
お送りいただいた写真のなかで、OLEDとは関係ないものもありますが、下記画像の矢印のものは半田が玉になって、うまく基板のパッドになじんでいないので、接触不良となりそうです。
今は導通していても時間がたつと不具合となりやすいので、基板に熱を伝えて、半田が山型になるように修正していただければ幸いです。

以上、お手数ですがご対応お願いいたします。

![KB61_OLEDfix](https://github.com/Yowkees/keyball/assets/84209772/858b13e7-5719-4393-9855-2d30826954c8)

**Reply by escape010** [reply]
回答ありがとうございます。
ご指摘いただ通り、ハンダ付を見直したところ左右とも、OLEDが点灯しました。
キーの入力も問題ありません。
ありがとうございました。

**Comment by Yowkees** [comment]
無事に直ったようで良かったです！
ぜひ末永くご愛用いただければ幸いです。
